index.html

```diff
@@ -27,6 +27,7 @@
         <meta name="apple-mobile-web-app-capable" content="yes">
         <title></title>
         <script src="dist/openmct.js"></script>
+        <script src="dist/CouchChangesFeed.js"></script>
```

Comment:
Let's load the worker code in the same way as [the sinewave generator does it](https://github.com/nasa/openmct/blob/master/example/generator/WorkerInterface.js#L31), so we don't have to impose any new markup on users.

Reply:
@akhenry Each Blob and the URL created from it will be a unique object, so I don't think that will work for SharedWorkers.
Let me see what else I can come up with though.

Reply:
This was resolved by adding an additional asset for the shared worker code and using the asset path to get the predictable URL for SharedWorker